src/cli.rs

```diff
@@ -367,6 +367,10 @@ Example: --print=Hb"
     #[clap(long)]
     pub curl_long: bool,
 
+    /// Generate shell completions or man pages.
+    #[arg(long, value_name = "KIND")]
```

Comment:
Can you add `hide_possible_values = true` and instead document it in long help? This allows us to preserve the current width of short help

```rs
    /// Generate shell completions or man pages.
    #[arg(
        long,
        value_name = "KIND",
        hide_possible_values = true,
        long_help = "\
Generate shell completions or man pages. Possible values are:

    complete-bash
    complete-elvish
    complete-fish
    complete-nushell
    complete-powershell
    complete-zsh
    man

Example: xh --generate=complete-bash > xh.bash"
    )]
    pub generate: Option<Generate>,
```

Reply:
Sure done in next commit, thanks!